docs/assets/files/Z_Bumpers.cpf

```diff
@@ -0,0 +1,12 @@
+@Controller Profile 2.0
+

```

Comment:
For a future troubleshooting benefit, I wonder if we should just add a comment here, to identify someone is using the correct file. Something simple like `# from smash64.online` or `# from nickthename`, I don't care. It doesn't seem to affect `Load Profile`.

Reply:
I believe these were created by Crovy although I'd need to check with him. Editing these files gives me the heebie-jeebies. I had to tell git to treat them as binaries to avoid it messing with the line endings, which I was worried might piss N-Rage off.